bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -551,10 +551,10 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- The default filenames are given relative to the root directory
      for the CLM2 data in the CESM distribution -->
 <!-- Plant function types (relative to {csmdata}) -->
-<paramfile phys="clm6_0" >lnd/clm2/paramdata/ctsm60_params.c240208.nc</paramfile>
-<paramfile phys="clm5_1" >lnd/clm2/paramdata/ctsm51_params.c240208.nc</paramfile>
-<paramfile phys="clm5_0" >lnd/clm2/paramdata/clm50_params.c240208.nc</paramfile>
-<paramfile phys="clm4_5" >lnd/clm2/paramdata/clm45_params.c240208.nc</paramfile>
+<paramfile phys="clm6_0" >/glade/work/samrabin/ctsm_fire-li2024-pr2_paramfiles/ctsm60_params.c240822.li2024crujra.nc</paramfile>
```

Comment:
We should move this into input data under lnd/clm2/paramdata and rimport it.

Reply:
Notes:
DONE `cp /glade/work/samrabin/ctsm_fire-li2024-pr2_paramfiles/ctsm60_params.c240822.nc /glade/campaign/cesm/cesmdata/inputdata/lnd/clm2/paramdata/.`
TODO
1) ./rimport -file
2) Update path and file name in namelist_defaults

Reply:
DONE